package.json

```diff
@@ -11,7 +11,8 @@
     "test": "nps test",
     "test:ts": "(tsc -p ./tsconfig.json && rimraf test-ts) || rimraf test-ts",
     "commitmsg": "opt --in commit-msg --exec \"validate-commit-msg\"",
-    "precommit": "lint-staged && opt --in pre-commit --exec \"npm start validate\""
+    "precommit": "lint-staged && opt --in pre-commit --exec \"npm start validate\"",
+    "doctoc": "doctoc --maxlevel 3 --title '**Table of Contents**' README.md"
```

Comment:
Any more specific reference? Not sure how would you want it to be, I thought https://github.com/paypal/glamorous/blob/b7424288431878bace8d9bb3b7d641e21a61b78f/package.json#L99-L104 would be good already?